spec/furniture/journal/entries_request_spec.rb

```diff
@@ -6,7 +6,17 @@
   let(:room) { journal.room }
   let(:member) { create(:person, spaces: [space]) }
 
-  describe "POST /entries" do
+  describe "#index" do
+    it "shows all the journal entries" do
+      sign_in(space, member)
+      entries = create_list(:journal_entry, 3, journal: journal)
```

Comment:
Would be nice to explicitly make sure in the test that we have both published and un-published entries and are displaying all of them. But not blocking to merge this in.

Reply:
Good call! I'll do that then merge!

Reply:
Thank you for adding that! ✅